src/pip/_internal/resolution/resolvelib/reporter.py

```diff
@@ -66,6 +66,7 @@ def starting_round(self, index: int) -> None:
 
     def ending_round(self, index: int, state: Any) -> None:
         logger.info("Reporter.ending_round(%r, state)", index)
```

Comment:
@pradyunsg I wasn't sure whether to make it conditional or replace the info level output.

Reply:
Making it conditional would be nicer.

Reply:
It's fine as is tho.